tests/src/rules/order.js

```diff
@@ -3218,39 +3218,6 @@ context('TypeScript', function () {
                 // { message: '`node:fs/promises` import should occur before import of `express`' },
               ],
             }),
-
-            test({
-              code: `
-                import express from 'express';
```

Comment:
let's keep this test case and update it so it matches the new behavior

Reply:
I've shifted it into valid - is that what you were thinking?